### PR TITLE
config: update CSP after react UI integration

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: {{cookiecutter.project_shortname}}/invenio.cfg [hunk 1]
```diff
@@ -47,11 +47,12 @@ APP_DEFAULT_SECURE_HEADERS = {
             "'self'",
             'fonts.googleapis.com',  # for fonts
             '*.gstatic.com',    # for fonts
+            'data:' # for fonts
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/invenio.cfg]:
```suggestion
            'data:', # for fonts
```

File: {{cookiecutter.project_shortname}}/invenio.cfg [hunk 2]
```diff
@@ -47,11 +47,12 @@ APP_DEFAULT_SECURE_HEADERS = {
...
             "'unsafe-inline'",  # for inline scripts and styles
+            "'unsafe-eval'", # for webpack build
             "blob:",            # for pdf preview
             # Add your own policies here (e.g. analytics)
-        ],
-    },
+    ],
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/invenio.cfg]: Shouldn't those 2 be kept?

```
    ],
},
```

File: {{cookiecutter.project_shortname}}/invenio.cfg [hunk 3]
```diff
@@ -47,11 +47,12 @@ APP_DEFAULT_SECURE_HEADERS = {
             "'self'",
             'fonts.googleapis.com',  # for fonts
             '*.gstatic.com',    # for fonts
+            'data:' # for fonts
             "'unsafe-inline'",  # for inline scripts and styles
+            "'unsafe-eval'", # for webpack build
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/invenio.cfg]: I noticed the CSP warning about this one, but I didn't see anything break from it. What was affected? (didn't see the `data` one though)

Comment (zzacharo) [reply {{cookiecutter.project_shortname}}/invenio.cfg]: this is webpack when it injects react in the dom and the data is when loading some fonts through the `data` scheme. Still the `unsafe-eval` is not a best practice and I saw people having issues due to webpack as it is using the method of string evaluation as javascript. We need to dig more into this. I will create an issue for that!

Comment (fenekku) [reply {{cookiecutter.project_shortname}}/invenio.cfg]: Yeah, I've been reluctant to allow the unsafe-eval. But webpack should be at build time. It shouldn't be on a served page right?

**Pull request merged**